modules/violation-view-policy-tab.adoc

```diff
@@ -9,21 +9,34 @@
 The *Policy* tab of the *Details* panel displays details of the policy that caused the violation.
 
 [discrete]
-== Policy Details section
+== Policy overview section
 
-The policy details section lists the following information:
+The policy overview section lists the following information:
 
-* *Id*: The numerical identifier for the policy.
-* *Name*: The name of the policy.
+* *Severity* - The risk level for the violation.
+* *Categories*: The policy category of the policy.
```

Comment:
This feels like a tautology but surprisingly we don't define the field in the policy docs.

Reply:
Just by looking at some examples, these seem like the default system RHACS policy categories (DevOps Best Practices, Security Best Practices, etc.). Would these also include ones that users created themselves in the Policy Management window?

Reply:
Yes it would be both.